packages/webapp/src/pages/delegates/index.tsx

```diff
@@ -22,80 +33,128 @@ interface Props {
 const MEMBERS_PAGE_SIZE = 18;
 
 export const DelegatesPage = (props: Props) => {
-    const { isError: isMembersDataFetchError, data: members } = useQuery({
+    const [ualAccount] = useUALAccount();
+    const accountName = ualAccount?.accountName;
+    const { data: currentMember } = useCurrentMember();
+
+    const { data: members } = useQuery({
         ...queryMembers(1, MEMBERS_PAGE_SIZE),
         keepPreviousData: true,
     });
 
-    const {
-        isError: isMemberStatsDataFetchError,
-        data: membersStats,
-    } = useQuery({
+    const loggedInMemberName = currentMember?.name || accountName;
+
+    const { data: membersStats } = useQuery({
         ...queryMembersStats,
         keepPreviousData: true,
     });
 
+    const { data: leadRepresentative } = useQuery({
+        ...queryHeadDelegate,
+        keepPreviousData: true,
+    });
+
+    const { data: myDelegation } = useQuery({
+        ...queryMyDelegation(members!, loggedInMemberName),
+        enabled: !!members && !!loggedInMemberName,
+        keepPreviousData: true,
+    });
+
+    if (
+        !loggedInMemberName ||
+        !members ||
+        !membersStats ||
+        !leadRepresentative ||
+        !myDelegation
+    ) {
+        return <Text size="lg">Fetching Data...</Text>;
+    }
+
+    const loggedInMember: MemberData | undefined =
+        members && getMemberRecordFromName(members, loggedInMemberName);
+    if (!loggedInMember) {
+        return <Text size="lg">Failed to get loggedInMember record</Text>;
+    }
+
     return (
         <RawLayout title="Election">
             <Text size="sm" className="mb-8">
                 Note: Data is in square brackets if it's not JSON (to show if
                 something's undefined)
             </Text>
+            <div>
+                <Text size="lg" className="bg-gray-200">
+                    My Delegation
+                </Text>
+                <Text size="sm" className="mt-4">
+                    {`You [${loggedInMember.account}] are level (rank) ${loggedInMember.election_rank} out of ${membersStats.ranks.length}`}
```

Comment:
When you're in JSX, you don't have to use template strings. You can just do:

`You [{loggedInMember.account}] are level (rank) {loggedInMember.election_rank} out of {membersStats.ranks.length}`

Reply:
I really like using tick quotes everywhere just to keep thing consistent (in case there's a special character like an apostrophe)... :)

Reply:
You're already not using tick quotes consistently in your JSX. Most strings in your JSX in this file are not wrapped in. `{` ... `}`. 

Besides that...apostrophes work fine in JSX as do most special characters...without the tick quotes. Using string template literals like this goes against React convention and is unnecessarily difficult to write and read.

This is a ReactNode. Basic strings are valid ReactNodes. And the convention is to use strings directly inside JSX. In fact, I'd argue on reason JSX is used in React is to emulate familiar HTML/XML structure. This moves us away from that familiarity.

1. I've never seen anyone do this purposefully in React, unless absolutely necessary.
1. This is one I do actually feel strongly about...hehe.
1. Sparky and I will not be doing this. Your code will be an outlier in this regard.

So I'd urge you to reconsider.